src/main/webapp/index.html

```diff
@@ -30,7 +30,7 @@
           <input id="destination-input" class="userInput" type="text" placeholder="Enter a destination location"/>
         </div>
         <div id="feature-buttons-container" class="flex-container">
-          <button class="feature-button" id="clean-air-button" alt="Find rest stops with clean air near this location">Clean air near here</button>
+          <button class="feature-button" id="clean-air-button" alt="Find rest stops with clean air near this location" onClick="showCleanAirMarkers()">Clean air near here</button>
```

Comment:
jfyi, update the css if you get time, http://screen/9EEAzvoEgMbqCXF, thought this button is the title of following options, but not big deal.

Reply:
Hmmmm ok, any suggestions for how to make it clearer? I was thinking of keeping it hidden unless a location is selected and there is data available, but I'm not sure how to make it distinct when it it shown